src/swagger.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Paying It Forward",
+    "description": "This is the documentation for Paying It Forward API. To access the paths that have a lock symbol next to it, you can send a signin request under Auth tab and copy the token from the `authorizationToken` response header and paste it in the `Authorize 🔒` section.",
```

Comment:
Well done, Ammar!! Everything looks good 👏👏

Just a comment regarding the main description, I guess it is enough to say "... To access the paths that have a lock symbol next to it, you need to sign in first."

<img width="1376" alt="image" src="https://user-images.githubusercontent.com/91065358/155883068-238f7dfd-75ac-4ae0-bb96-c22d1b901cf8.png">

Reply:
It is not necessary to copy and paste authorization token. After signing in you can access locked endpoints automatically.

Reply:
Yeah you are right, it seems that it is using the browser's cookies
I will edit the description to what you suggested, and I will remove the swagger token from everywhere